app/views/smart_answers/landing.html.erb

```diff
@@ -1,17 +1,34 @@
 <% start_node = @presenter.start_node %>
 
-<% if start_node.meta_description.present? %>
-  <% content_for :head do %>
+<% content_for :head do %>
+  <% if start_node.meta_description.present? %>
     <meta name="description" content="<%= start_node.meta_description %>">
   <% end %>
+  <% if @content_item.present? %>
+    <%= render 'govuk_publishing_components/components/machine_readable_metadata',
+      schema: :article,
+      content_item: @content_item %>
+    <%= render 'govuk_publishing_components/components/machine_readable_metadata',
+      schema: :government_service,
+      content_item: @content_item %>
```

Comment:
Nit: Its strange, we end up with duplicate twitter meta tags 
```html
<meta name="twitter:card" content="summary">
<meta name="twitter:card" content="summary">
```

Although this pre-exists in Frontend too and probably doesn't cause a problem. But just looked odd.